dashboard/models/alert.py

```diff
@@ -313,7 +314,10 @@ def alert_firefox_out_of_date(self):
 
     def _firefox_info(self):
         release_json = requests.get('https://product-details.mozilla.org/1.0/firefox_versions.json')
-        return release_json.json()
+        if release_json.status_code == 200:
```

Comment:
nit, you could use `release_json.ok is True` so that it includes all conditions where the request succeeds (http://docs.python-requests.org/en/master/api/#requests.Response.ok) (note that its different from `requests.code.ok`)

Reply:
Should we just capture this as an improvement?  Or block on it.  Seems like functionally it does almost the same thing.

Reply:
its really quick to fix - i wouldnt capture this if you dont do it though, its just a nicer pattern

Reply:
Did some more thinking on this:

if json parsing fails here things will still fail (i ran into that sometimes)
a possible additional safety is something like that:

```
if release_json.ok:
  try:
     return json.loads(release_json.text)
  except Exception as e:
    logger.warning("failed to parse json: {}".format(e))
    return None
```

am happy to pr that in or something similar